lib/topological_inventory/ansible_tower/operations/core/ansible_tower_client.rb

```diff
@@ -120,11 +120,31 @@ def verify_ssl_mode
             default_endpoint.verify_ssl ? OpenSSL::SSL::VERIFY_PEER : OpenSSL::SSL::VERIFY_NONE
           end
 
+          # TODO: Join with availability_check methods
           def ansible_tower
             @ansible_tower ||= connection_manager.connect(
-              default_endpoint.host, authentication.username, authentication.password, :verify_ssl => verify_ssl_mode
+              :base_url       => full_hostname(default_endpoint),
+              :username       => authentication.try(:username),
+              :password       => authentication.try(:password),
+              :verify_ssl     => verify_ssl_mode,
+              :receptor_node  => default_endpoint.receptor_node.to_s.strip,
+              :account_number => account_number
             )
           end
+
+          def account_number
```

Comment:
@lindgrenj6 I need to discuss this method. 

Can I expect that forwardable_headers from catalog will contain account_number?

Reply:
Yes - it will be in the x-rh-identity header.